go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/prometheus/client_golang v1.7.1
 	github.com/prometheus/client_model v0.2.0
 	github.com/prometheus/common v0.15.0
+	github.com/prometheus/exporter-toolkit v0.5.1-0.20210113141400-89922ed1bafb
```

Comment:
why do we have the timestamp and the commit hash?

Can we `go get @v0.5.1` again?